Mathlib/CategoryTheory/Monoidal/ExternalProduct.lean

```diff
@@ -0,0 +1,98 @@
+/-
+Copyright (c) 2025 Robin Carlier. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Robin Carlier
+-/
+import Mathlib.CategoryTheory.Monoidal.FunctorCategory
+import Mathlib.CategoryTheory.Functor.Currying
+
+/-!
+# External product of diagrams in a monoidal category
+
+In a monoidal category `C`, given a pair of diagrams `K₁ : J₁ ⥤ C` and `K₂ : J₂ ⥤ C`, we
+introduce the external product `K₁ ⊠ K₂ : J₁ × J₂ ⥤ C` as the bifunctor `(j₁, j₂) ↦ K₁ j₁ ⊗ K₂ j₂`.
+The notation `- ⊠ -` is scoped to `MonoidalCategory.ExternalProduct`.
+-/
+
+universe v₁ v₂ v₃ v₄ u₁ u₂ u₃ u₄
+
+namespace CategoryTheory.MonoidalCategory
+
+variable (J₁ : Type u₁) (J₂ : Type u₂) (C : Type u₃)
+    [Category.{v₁} J₁] [Category.{v₂} J₂] [Category.{v₃} C] [MonoidalCategory C]
+
+/-- The (uncurried version of the) external product bifunctor: Given diagrams
```

Comment:
It seems to me that what you call "uncurried" is what we call "curried". Could you update the docstrings and the names accordingly?